### ISSUE TITLE
Inappropriate HTML Render

### ISSUE THREAD

**Issue by tbhaxor** [Leecason/element-tiptap#77]
When the text is aligned centered, the editor should apply some inline styles like `style='text-align: center'` but its using data-* 

```html
<p><strong>This</strong></p><p></p><h1>ddddd</h1><p></p><p data-text-align="center">{{code}}</p>
```

Due to this the HTML in email not rendered properly

**Comment by tbhaxor** [comment]
**EMAIL RECEIVED**
![image](https://user-images.githubusercontent.com/28386721/81970708-937f8200-963d-11ea-8925-7d092fe05cef.png)


**TEMPLATE**
![image](https://user-images.githubusercontent.com/28386721/81970789-b0b45080-963d-11ea-9a8a-92aaf4b5c687.png)

PS: focus on LOL and COD text

**Comment by Leecason** [comment]
fixed in 1.21.0
